packages/reaction-core/client/helpers/globals.js

```diff
@@ -72,48 +72,104 @@ this.getProductsByTag = function (tag) {
 };
 
 /**
- * maybeDeleteProduct
- * @summary confirm product deletion, delete, and alert
- * @todo - refactor this back into templates. this is bad.
- * @param {Object} product - product Object
- * @return {Object} - returns nothing, and alerts,happen here
+ * publishProduct
+ * @summary product publishing and alert
+ * @param {Object} productOrArray - product Object
+ * @returns {Function}
  */
-this.maybeDeleteProduct = function (product) {
-  let productIds;
-  let title;
-  let confirmTitle = "Delete this product?";
-
-  if (_.isArray(product)) {
-    if (product.length === 1) {
-      title = product[0].title || "the product";
-      productIds = [product[0]._id];
-    } else {
-      title = "the selected products";
-      confirmTitle = "Delete selected products?";
+this.publishProduct = function (productOrArray) {
```

Comment:
should this just be `publishProduct` and not `this.publishProduct` where `this.publishProduct` is window scope, and `publishProduct` should be package scoped.